meorg_client/client.py

```diff
@@ -25,13 +27,20 @@ def __init__(self, base_url, email=None, password=None):
         """
         self.base_url = base_url
```

Comment:
Can we have a default value for base_url. Usually it doesn't change

Reply:
Setting the default to the production URL.